home-assistant-tmpfile.conf

```diff
@@ -0,0 +1 @@
+d /var/lib/hass 0755 hass hass -
```

Comment:
/var/lib/hass is not a place for tmp files only, configuration files are also stored in this directory. Are you sure this is the right way to declare this kind of directory ?

Reply:
I had a quick chat with the other TUs on how to solve this directory issue with user permissions, and that seems to be the way to do it. It is pretty nice and should work, even though the name is misleading.